### PR TITLE
change README

### PR DESCRIPTION
**Pull request by TacticalCrab**
Zaktualizowano README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,4 @@
-<p align="center">
-  <a href="http://nestjs.com/" target="blank"><img src="https://nestjs.com/img/logo-small.svg" width="120" alt="Nest Logo" /></a>
-</p>
-
-[circleci-image]: https://img.shields.io/circleci/build/github/nestjs/nest/master?token=abc123def456
-[circleci-url]: https://circleci.com/gh/nestjs/nest
-  
-  <p align="center">A progressive <a href="http://nodejs.org" target="_blank">Node.js</a> framework for building efficient and scalable server-side applications.</p>
+<p>RETROMACHINA DOBRA APKA :)</p>
```
Comment (DiD3n) [inline README.md]: use readme header
ex.
```md
# RETROMACHINA DOBRA APKA :)
```
docs:
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

Comment (DiD3n) [reply README.md]: md*

Comment (JUtupe) [inline README.md]: Aplikacja wyglądałaby tu lepiej

**Review by JUtupe — CHANGES_REQUESTED** [review]

**Review by DiD3n — APPROVED** [review]
looks legit now

**Pull request merged**